soccer/gameplay/tactics/positions/goalie.py

```diff
@@ -121,7 +121,7 @@ def execute_running(self):
         if self.robot != None:
             #face the ball in all states but intercept
             if not self.has_subbehavior_with_name('intercept'):
-                self.robot.face(main.ball().pos)
+                pass#self.robot.face(main.ball().pos)
```

Comment:
Do we want to assume motion control is getting fixed enough that we should just add this back in?

Reply:
TODO it, I think